lib/elixir/lib/port.ex

```diff
@@ -88,10 +123,13 @@ defmodule Port do
   end
 
   @doc """
-  See [`:erlang.ports/0`](http://www.erlang.org/doc/man/erlang.html#ports-0)
+  Returns a list of the ports of the current node.
```

Comment:
Yes, for sure. English is not my native langage and "of"/"for" is a common confusion I still make.